prompting/datasets/wiki.py

```diff
@@ -265,22 +213,26 @@ class WikiDateDataset(Dataset):
         "November",
         "December",
     )
-    EXCLUDE_CATEGORIES = ("articles", "wiki", "pages", "cs1")
-
-    def __init__(self, max_tries: int = 10, seed=None):
-        self.max_tries = max_tries
-        self.seed = seed
-        self.rng = random.Random(seed)
-        
-    def extract_dates_and_sentences(self, text: str) -> Tuple[str, str]:
+    EXCLUDE_CATEGORIES: tuple = ("articles", "wikipedia", "pages", "cs1")
+    max_tries: int = 10
+    seed: int | None = None
```

Comment:
Let's not use the `|` operator for typing and instead use `Optional[int]` unless there's a good reason for this.

Reply:
as per discussion, typing library is deprecated since py3.9